### PR TITLE
Update repo/docs branding in line with new website

### PR DESCRIPTION
**Pull request by jumaffre**
Branding is now in line with https://ccf.microsoft.com

~Waiting for SVG files before merging.~ Done! Thank you @shokouedamsr

### CODE REVIEW

File: doc/overview/what_is_ccf.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ This page gives a broad overview of the fundamentals of CCF. This is where you s
 CCF in a Hundred Words
 ----------------------
 
-The Confidential Consortium Framework (CCF) is an open-source framework for building a new category of secure, highly-available, and performant applications that focus on multi-party compute and data.
+The `Confidential Consortium Framework (CCF) <https://ccf.microsoft.com/>`_ is an open-source framework for building a new category of secure, highly-available, and performant applications that focus on multi-party compute and data.
```
Comment (achamayou) [inline doc/overview/what_is_ccf.rst:9]:
```suggestion
The `Confidential Consortium Framework (CCF) <https://ccf.dev/>`_ is an open-source framework for building a new category of secure, highly-available, and performant applications that focus on multi-party compute and data.
```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
+<img alt="ccf" align="right" src="doc/_static/ccf.png" width="200">
 
 - Continuous Build: [![Build Status](https://dev.azure.com/MSRC-CCF/CCF/_apis/build/status/CCF%20Github%20CI?branchName=main)](https://dev.azure.com/MSRC-CCF/CCF/_build/latest?definitionId=3&branchName=main)
 - Daily Build: [![Build Status](https://dev.azure.com/MSRC-CCF/CCF/_apis/build/status/CCF%20GitHub%20Daily?branchName=main)](https://dev.azure.com/MSRC-CCF/CCF/_build/latest?definitionId=7&branchName=main)
 - Doc Build: [![docs](https://dev.azure.com/MSRC-CCF/CCF/_apis/build/status/CCF%20GitHub%20Pages?branchName=main)](https://dev.azure.com/MSRC-CCF/CCF/_build/latest?definitionId=4&branchName=main)
 - Containers: [![Build and Publish Release Containers](https://github.com/microsoft/CCF/actions/workflows/containers.yml/badge.svg)](https://github.com/microsoft/CCF/actions/workflows/containers.yml)
 
-The Confidential Consortium Framework (CCF) is an open-source framework for building a new category of secure, highly available,
+The [Confidential Consortium Framework (CCF)](https://ccf.microsoft.com/) is an open-source framework for building a new category of secure, highly available,
```
Comment (achamayou) [inline README.md:10]:
```suggestion
The [Confidential Consortium Framework (CCF)](https://ccf.dev/) is an open-source framework for building a new category of secure, highly available,
```

File: doc/overview/what_is_ccf.rst (hunk 1 above)
Comment (jumaffre) [reply doc/overview/what_is_ccf.rst:9]: `ccf.dev` automatically redirects to `ccf.microsoft.com` so it feels more natural to refer to the final URL instead?

**Review by eddyashton — APPROVED** [review]

File: doc/overview/what_is_ccf.rst (hunk 1 above)
Comment (achamayou) [reply doc/overview/what_is_ccf.rst:9]: It does now, but that's an implementation detail. The person who did the website also suggested it was better for SEO to use ccf.dev.

**Review by achamayou — APPROVED** [review]

**Pull request merged**